### PR TITLE
SaveAccount without LoadAccount first

### PR DESCRIPTION
**Pull request by BeniaminDrasovean**
Add the dataTrie to tries cacher if SaveAccount is called, but the dataTrie is not present in the hasher.

### CODE REVIEW

**Review by SebastianMarian — DISMISSED** [review]

File: data/state/accountsDB.go [hunk 1]
```diff
@@ -442,6 +442,10 @@ func (adb *AccountsDB) saveDataTrie(accountHandler baseAccountHandler) error {
 		"new root hash", accountHandler.GetRootHash(),
 	)
 
+	if adb.dataTries.Get(accountHandler.AddressBytes()) == nil {
```
Comment (SebastianMarian) [inline data/state/accountsDB.go:445]: check.IfNil ?

Comment (SebastianMarian) [reply data/state/accountsDB.go:445]: if Get returns an interface

Comment (SebastianMarian) [inline data/state/accountsDB.go:445]: check.IfNil if Get returns an interface

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System Test passed

**Pull request merged**